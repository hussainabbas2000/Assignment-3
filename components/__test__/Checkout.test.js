/**
 * @jest-environment jsdom
 */
 import React from "react";
 import ReactDOM  from "react-dom";
 import Checkout from "../../pages/checkout";
 import {render,cleanup} from '@testing-library/react';
 import "@testing-library/jest-dom/extend-expect"
 import renderer from "react-test-renderer";
 afterEach(cleanup);
 it("renders checkout without crash",()=>{
     const div = document.createElement("div");
     ReactDOM.render(<Checkout></Checkout>,div); 
 })
 it("renders checkout correctly",()=>{
     const {getByTestId} = render(<Checkout/>)
     const box = getByTestId("checkout")
     expect(box).not.toHaveClass()
 
 })
 
 it("matches snapshot 6",()=>{
     const item = renderer.create(<Checkout></Checkout>).toJSON();
     expect(item).toMatchSnapshot();
 })