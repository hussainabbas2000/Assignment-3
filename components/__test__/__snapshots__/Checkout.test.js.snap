// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 6 1`] = `
<div
  data-testid="checkout"
>
  <div
    className="MuiContainer-root MuiContainer-maxWidthLg"
  >
    <div
      className="makeStyles-main-1"
    >
      <h1
        className="MuiTypography-root MuiTypography-h4 MuiTypography-colorPrimary MuiTypography-gutterBottom"
      >
        Coming soon!
      </h1>
    </div>
  </div>
</div>
`;
